src/blueapi/config.py

```diff
@@ -39,9 +39,17 @@ class StompConfig(BlueapiBaseModel):
     Config for connecting to stomp broker
     """
 
-    host: str = "localhost"
-    port: int = 61613
-    auth: BasicAuthentication | None = None
+    enabled: bool = Field(
+        description="True if blueapi should connect to stomp for asynchronous "
+        "event publishing",
+        default=False,
+    )
+    host: str = Field(description="STOMP broker host", default="localhost")
```

Comment:
nit: We could also use

```suggestion
    host: Annotated[str, "STOMP broker host"] = "localhost"
```

but I think it's a matter of personal taste.

Reply:
Would be good to standardise across the repo though, maybe a different PR?

Reply:
Fine with that, looks like our use of Annotated is very small, but is used for fastapi Depends and for one of the SkipJsonSchemas